src/plugin/ipc/connectionlist.cpp

```diff
@@ -488,24 +490,23 @@ void ConnectionList::sendReceiveMissingFds()
     }
   }
 
-  fd_set rfds;
-  fd_set wfds;
   int restoreFd = protectedFd();
   size_t numOutgoingCons = outgoingCons.size();
   while (numOutgoingCons > 0 || numIncomingCons > 0) {
-    FD_ZERO(&wfds);
+    struct pollfd socketFd = {0};
     if (outgoingCons.size() > 0) {
-      FD_SET(restoreFd, &wfds);
+      socketFd.fd = restoreFd;
+      socketFd.events = POLLOUT;
     }
-    FD_ZERO(&rfds);
     if (numIncomingCons > 0) {
-      FD_SET(restoreFd, &rfds);
+      socketFd.fd = restoreFd;
+      socketFd.events |= POLLIN;
     }
 
-    int ret = _real_select(restoreFd+1, &rfds, &wfds, NULL, NULL);
+    int ret = _real_poll (&socketFd, 1, -1);
```

Comment:
I thought we were favoring a style with no space between the function name and the opening paenthesis for the arguments:
  _real_poll (&socketFd, 1, -1)  -> _real_poll(&socketFd, 1, -1)